libr/bin/bfile.c

```diff
@@ -408,7 +406,7 @@ R_API bool r_bin_file_object_new_from_xtr_data(RBin *bin, RBinFile *bf, ut64 bas
 	ut64 sz = data->size;
 
 	RBinPlugin *plugin = get_plugin_with_buffer (bin, data->buf);
-	bf->buf = r_buf_new_with_bufref (data->buf);
+	bf->buf = r_buf_ref (data->buf);
```

Comment:
If this method returns a new instance it should be named “new” if its just increasing the ref ignore this msg

Reply:
it is just increasing the ref.